lib/routes/root.js

```diff
@@ -41,12 +41,13 @@ module.exports = {
     }
 
     // figure it out from .git
-    var gitDir = path.resolve(__dirname, '..', '..', '.git');
-    exec('git rev-parse HEAD', { cwd: gitDir }, function(err, stdout) { // eslint-disable-line handle-callback-err
+    const gitDir = path.resolve(__dirname, '..', '..', '.git');
+    exec('git rev-parse HEAD', { cwd: gitDir }, (err, stdout) => { // eslint-disable-line handle-callback-err
       commitHash = stdout.replace(/\s+/, '');
-      var configPath = path.join(gitDir, 'config');
-      var cmd = 'git config --get remote.origin.url';
-      exec(cmd, { env: { GIT_CONFIG: configPath } }, function(err, stdout) { // eslint-disable-line handle-callback-err
+      const configPath = path.join(gitDir, 'config');
+      const cmd = 'git config --get remote.origin.url';
+      const env = Object.assign({}, process.env, { GIT_CONFIG: configPath });
+      exec(cmd, { env }, (err, stdout) => { // eslint-disable-line handle-callback-err
```

Comment:
For unknown reasons, the move to nyc meant that this started failing on my local machine.  I found I had to explicitly pass in `process.env` in order for it to locate the `git` executable correctly.